app/src/main/res/layout/navigation_bar_navigation.xml

```diff
@@ -154,4 +152,26 @@
             android:tooltipText="@string/hamburger_menu_tooltip"
             app:privateMode="@{viewmodel.isPrivateSession}"/>
     </LinearLayout>
+
+    <!-- Horizontal Divider -->
+        <FrameLayout
+            android:clickable="true"
+            android:id="@+id/moveBar"
+            android:radius="4dp"
+            android:layout_margin="10dp"
+            android:layout_width="128dp"
+            android:layout_height="32dp"
+            android:tooltipText="@string/move_tooltip"
+            android:layout_gravity="center_horizontal"
+            app:visibleGone="@{viewmodel.isCurved}">
+
+            <View
```

Comment:
We should add `android:duplicateParentState="true"` to this view so it gets the hover state from its parent.

Reply:
good point